deploy/olm-catalog/akka-cluster-operator/0.2.3/akka-cluster-operator-certified.v0.2.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+      mediatype: image/svg+xml
+  install:
+    spec:
+      deployments:
+        - name: akka-cluster-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: akka-cluster-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: akka-cluster-operator
+              spec:
+                containers:
+                  - command:
+                      - akka-cluster-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: akka-cluster-operator
+                      - name: RELATED_IMAGE_CONTROLLER
+                        value: registry.connect.redhat.com/lightbend/akkacluster-operator:v0.2.2   
+                    image: registry.connect.redhat.com/lightbend/akka-cluster-operator-certified:0.2.2
```

Comment:
And what about these?

Reply:
Same, the operator image is not changing.